doc/api/errors.md

```diff
@@ -238,7 +238,7 @@ function MyError() {
 
 // Without passing MyError to captureStackTrace, the MyError
 // frame would show up in the .stack property. By passing
-// the constructor, we omit that frame and all frames above it.
+// the constructor, we omit that frame, and retain all frames above it.
```

Comment:
This changes the meaning of this sentence now, you would want to replace “above” with “below” if you talk about the retained frames